en/docs/includes/deploy/assign-custom-hostname.md

```diff
@@ -18,13 +18,13 @@
 
 3.  Click on the API to edit its configurations.
 
-     [![Edit the API](/../../../../assets/img/includes/deploy/select-api.png)](/../../../../assets/img/includes/deploy/select-api.png)
+     [![Edit the API](../../assets/img/includes/deploy/select-api.png)](../../assets/img/includes/deploy/select-api.png)
 
 4.  Click **Deployments**.
 
 5.  Select the newly created Gateway environment, Virtual Host (foods.com), and a Revision.
 
-     [Create a new revision](/../../../../design/create-api/create-api-revisions) if no revisions exist.
+     [Create a new revision](../../design/create-api/create-api-revisions.md) if no revisions exist.

```

Comment:
This change is wrong pls revert it.

Reply:
reverted with https://github.com/wso2/docs-apim/pull/3842/commits/013b5cef34ccd5cbfe5edccf52d59d8a67dbb6dd